sdl2promela/sdl/model.py

```diff
@@ -221,8 +222,12 @@ class Parameter:
     target_variable: VariableReference
     """Target variable reference."""
 
+    declared_type: object
```

Comment:
I started the use of `object` for types from opengeode, but after some time I think that the correct one is `Any` https://docs.python.org/3/library/typing.html#typing.Any